### ISSUE TITLE
C++ indexer depends on @rust_linux_x86_64//:rustc_lib

### ISSUE THREAD

**Issue by ivan444** [kythe/kythe#5371]
Hi!

Our CI (at https://github.com/chipsalliance/verible; Bazel based) is running Kythe C++ indexer to update our project at http://cs.opensource.google .

The CI downloads Kythe release and does Bazel build with `extractors.bazelrc`, which now fails as we hit runtime dependency on `@rust_linux_x86_64//:rustc_lib`.

Could you make that dependency optional? I.e. only when processing rust code?

Our workaround is to fork `extractors.bazelrc` and drop the languages we don't care about. Works, but it's a bit fragile.

**Comment by shahms** [comment]
Yes; the C++ indexer should not depend on Rust.

**Comment by shahms** [comment]
I think the issue may be that we just run `build` with all of the known extractors in `extractrors.bazelrc`

I'm not sure there's an easy way around that and may require a bit of work (if that's the cause)

**Comment by ivan444** [comment]
Yep, the issue is that `extractors.bazelrc` adds all known extractors to build.

A simple solution is to change the instructions:
- Create slimmer bazelrc (no language specific `--experimental_action_listener`s)
- Tell people to manually add `--experimental_action_listener=` to their bazel invocation
